packages/@ember/-internals/glimmer/tests/integration/application/debug-render-tree-test.ts

```diff
@@ -811,7 +830,7 @@ if (ENV._DEBUG_RENDER_TREE) {
             name: 'hello-world',
             args: { positional: [], named: { name: 'first' } },
             instance: null,
-            template: 'my-app/templates/components/hello-world.hbs',
```

Comment:
this field is based of `moduleName`, which we've talked about getting rid of. 

It still works when a template is resolved, but when using `setComponentTemplate`, there is no module name to associate -- could come from anywhere